notification-support/src/main/resources/avro/stream-registry-notification.avdl

```diff
@@ -2,7 +2,7 @@
 protocol StreamRegistry {
 
```

Comment:
@teabot @nahguam Can you provide feedback about this please...
It looks reasonable for me

Reply:
As you can see `AvroStream` includes the `schemaKey`. I noticed that `AvroStreamBinding` doesn't have the `streamKey` even though it can be build with: 
```
streamDomain
streamName
streamVersion
```
This key is really useful for downstream consumers because it makes posible to perform stateful operations (i.e. an aggregation). With a key the consumers doesn't have to guess how they have to build one.

Can you include it and change the name according with `AvroSchema` and `AvroStream`, i.e.:
```
  record AvroStreamBinding {
    string domain;
    string name;
    int version;
    string infrastructureZone;
    string infrastructureName;
    string description;
    array<Tag> tags;
    string type;
    string configurationString;
    union {null, string} statusString = null;
    union {null, AvroKey} streamKey = null;
  }
```
And validate it is coherent with its stream

Reply:
`streamKey` can be composed from:

```
    string domain;
    string name;
    int version;
```

same for `infrastructureKey`. I think we should include the keys or the flattened component parts, not both.

Reply:
done